### PR TITLE
[fix] Construct image form 2d torch tensor

### PR DESCRIPTION
**Pull request by avkudr**
This PR provides a fix for `aim.Image` construction form `torch.tensor`

Constructing an image from a tensor with dimensions `(2,2)`, that corresponds to a greyscale image, was causing an exception

PR contains:
- failing unit test
- and the fix itself

### CODE REVIEW

**Review by alberttorosyan — COMMENTED** [review]
@avkudr looks good!
should the similar fix be applied to image built from `tf.Tensor()`?

**Comment by avkudr** [comment]
@alberttorosyan good catch! There was also a problem with `tf` implementation: casting to `int8` instead of `uint8`.

Note the order of channels for image representation is different in pytorch and tensorflow:
- pytorch: `[batch_size, channels, height, width]` [(link)](https://pytorch.org/vision/stable/transforms.html?highlight=normalize)
- tf: `[batch_size, height, width, channels]` [(link)](https://www.tensorflow.org/hub/common_signatures/images)

New commit includes:
- fix for `tf.tensor`
- unit test for `tf` => `aim.Image` conversion
- fixes of code style issues

File: tests/sdk/test_image_construction.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    # greyscale, dims = (2,2)
+    img2 = np.array([[0, 1], [1, 0]], np.uint8)
+    # RGB, dims = (3,2,2)
+    img3 = np.array([[[0, 1], [1, 0]], [[0, 1], [1, 0]], [[0, 1], [1, 0]]], np.uint8)
+
+    def test_image_from_torch_tensor(self):
+        try:
+            import torch
+        except (ImportError, AssertionError):
+            raise ValueError('Cannot convert from torch.Tensor')
```
Comment (alberttorosyan) [inline tests/sdk/test_image_construction.py:19]: so this tests are failing if tensorflow/pytorch are not installed. need to add both in `requirements.dev.txt`

**Review by alberttorosyan — CHANGES_REQUESTED** [review]
Just noticed that workflow is failing due to missing dependencies.

File: tests/sdk/test_image_construction.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+        self.assertEqual(Image(torch.tensor(self.img1.astype(np.float32))), Image(255 * np.transpose(self.img1, (1, 2, 0))))
+        self.assertEqual(Image(torch.tensor(self.img2.astype(np.float32))), Image(255 * self.img2))
+        self.assertEqual(Image(torch.tensor(self.img3.astype(np.float32))), Image(255 * np.transpose(self.img3, (1, 2, 0))))
+
+    def test_image_from_tf_tensor(self):
+        try:
+            import tensorflow as tf
+        except (ImportError, AssertionError):
+            raise ValueError('Cannot convert from torch.Tensor')
```
[the remark this answers is not in the document]
Comment (avkudr) [reply tests/sdk/test_image_construction.py]: done

File: tests/sdk/test_image_construction.py (hunk 1 above)
Comment (avkudr) [reply tests/sdk/test_image_construction.py:19]: done

**Comment by avkudr** [comment]
@alberttorosyan should be all good now

**Comment by alberttorosyan** [comment]
@avkudr looks great, thanks for contribution!
merging this PR.

**Comment by alberttorosyan** [comment]
resolves #1234
